arm-operationalinsights/2015-03-20/swagger/OperationalInsights.json

```diff
@@ -0,0 +1,1146 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Operational InsightsManagement Client for LinkTargets, StorageInsightConfigs and SavedSearches.",
+    "description": ".Net client wrapper for the REST API for Azure Operational Insights",
+    "version": "2015-03-20"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
+  ],
+  "produces": [
+    "application/json",
+    "text/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/storageInsightConfigs/{name}": {
+      "put": {
+        "tags": [
+          "StorageInsights"
+        ],
+        "operationId": "StorageInsights_CreateOrUpdate",
+        "description": "Create or update a storage insight.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource group name of the storage insight."
```

Comment:
Resource Group name of the Log Analytics Workspace (since this is the parent resource of storageInsightsConfigs resource and that resource can't live on its own)